.github/workflows/pr-e2e-testing.yaml

```
@@ -17,17 +17,40 @@ on:
-  e2e:
+  e2e-tests:
     runs-on: "ubuntu-24.04"
+    # =================================================================
+    # This matrix will create 4 independent jobs that run in parallel.
+    # Each job will execute all the steps below.
+    # =================================================================
+    strategy:
+      fail-fast: false
+      matrix:
+        ginkgo_node: [1, 2, 3, 4]
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)